packages/tools/src/utilities/math/ellipse/pointInEllipse.ts

```diff
@@ -1,38 +1,38 @@
-import type { Types } from '@cornerstonejs/core';
-
-type Ellipse = {
-  center: Types.Point3;
-  xRadius: number;
-  yRadius: number;
-  zRadius: number;
-};
-
 /**
  * Given an ellipse and a point, return true if the point is inside the ellipse
  * @param ellipse - The ellipse object to check against.
  * @param pointLPS - The point in LPS space to test.
  * @returns A boolean value.
  */
-export default function pointInEllipse(
-  ellipse: Ellipse,
-  pointLPS: Types.Point3
-): boolean {
-  const { center: circleCenterWorld, xRadius, yRadius, zRadius } = ellipse;
-  const [x, y, z] = pointLPS;
-  const [x0, y0, z0] = circleCenterWorld;
+export default function pointInEllipse(ellipse, pointLPS, inverts) {
+  const { center, xRadius, yRadius, zRadius } = ellipse;
 
-  let inside = 0;
-  if (xRadius !== 0) {
-    inside += ((x - x0) * (x - x0)) / (xRadius * xRadius);
+  let invXRadiusSqToUse, invYRadiusSqToUse, invZRadiusSqToUse;
```

Comment:
Make these constants directly extracted from the inverts object, but have updated the inverts object to have the default values first if not provided.